Scripts/Python/stupStartUp.py

```diff
@@ -104,3 +108,20 @@ def OnServerInitComplete(self):
             PtShowDialog("GUIDialog04b")
         else:
             PtShowDialog("GUIDialog06")
+
+        self.randomizeBackground()
+
+    
+    ###########################
+    def randomizeBackground(self):
+        if not BGObj.sceneobject or not BGLayer.layer:
+            PtDebugPrint("stupStartUp: Missing ptAttribs for randomized backgrounds.  Leaving default...") 
+            return
+
+        # Get the first available ImageLibModifier on our Background SceneObject
+        if ilm := BGObj.sceneobject.getImageLibMods()[0]:
+            # Choose a random image from the available images
+            bgChoice = random.choice(ilm.getImages())
+            # Update the Background Layer's texture to our chosen image
+            BGLayer.layer.texture = bgChoice
```

Comment:
I wonder if it would be better for there to be separate `ptLayer.getTexture()` and `ptLayer.setTexture()` methods to match the rest of the API?

Reply:
I was aiming for it to be Pythonic as a new addition, but I can split those back out if we want it to be consistent (which definitely makes sense).